electrumsv/wallet_database/functions.py

```diff
@@ -1487,31 +1483,33 @@ def _write(db: Optional[sqlite3.Connection]=None) -> bool:
     return db_context.post_to_thread(_write)
 
 
-# TODO(1.4.0) Reorgs. Remove when we have replaced with a reference server equivalent.
-# def set_transactions_reorged(db_context: DatabaseContext, tx_hashes: list[bytes]) \
-#         -> concurrent.futures.Future[bool]:
-#     """
-#     Reset transactions back to unverified state as a batch.
-
-#     NOTE This may not restore the correct block height, which is prohibitive. 0 is unconfirmed,
-#     and -1 is unconfirmed parents. We do not have the information to know if it has unconfirmed
-#     parents.
-#     """
-#     timestamp = get_posix_timestamp()
-#     sql = (
-#         "UPDATE Transactions "
-#         "SET date_updated=?, flags=(flags&?)|?, block_hash=NULL, block_position=NULL, "
-#             "fee_value=NULL, proof_data=NULL "
-#         "WHERE tx_hash IN ({})")
-#     sql_values = [ timestamp, ~TxFlags.MASK_STATE, TxFlags.STATE_CLEARED ]
-
-#     def _write(db: Optional[sqlite3.Connection]=None) -> bool:
-#         rows_updated = execute_sql_by_id(db, sql, sql_values, tx_hashes)[0]
-#         if rows_updated < len(tx_hashes):
-#             # Rollback the database transaction (nothing to rollback but upholding the convention)
-#             raise DatabaseUpdateError("Rollback as nothing updated")
-#         return True
-#     return db_context.post_to_thread(_write)
+# TODO(1.4.0) - There are currently no tasks or initial startup procedures that check for
```

Comment:
This should be incorrect, the spent outputs stuff via `read_spent_outputs_to_monitor` should handle this IIUC.